### ISSUE TITLE
[suggestion] remove vote button for non-admins.

### ISSUE THREAD

**Issue by TheDavestDave** [VOREStation/VOREStation#4859]
#### Brief description of the issue
I don't really see a use for it outside of admin events fucking up the station, and if that happens there's admins online. and the few examples I can think off is a engine mistake. I think that could be fixed by re-working bridge so that the evac console and captain's ID are upstairs away form the SM explosion. Allowing people to break in and steal the captian's spare ID if no command are online.

As it is there's no reasons to use it, so it's liable to be used if some well meaning idiot thinks no power and unable to do any job is a good reason to start a vote, it can pass because people don't vote against it for what ever reason, even if most people don't want to call tram.

I know a lot of people who pay attenion here play more here and for longer than I have, has it even even been used for it's intended purpose?  There's also no information on what is a good reason to call a vote. This is all I could find via google
https://wiki.vore-station.net/Commands#Vote 
"Vote

Used to start a vote or to vote if a vote was started.
There are two kinds of votes:

    Restart: If more players vote for yes then for no, the round will be restarted.
    Change game mode: You can vote for a game mode. If the majority of the players vote for one game mode the round will restart and load this game mode. If no majority is formed, the vote will leave the game as it is (No Change). If the majority votes equally for multiple game modes, this will create a tie and the game mode is randomly chosen between those. Change mode votes are disabled during the round."

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)
 6c758051139daed75163f1941b40600bd97dceaa 
)

tl;dr
I don't see a reason the vote command would ever be needed.
It's very easy to misuse if there's no admins online
There's no information to when's a good reason to use it

**Comment by Mewchild** [comment]
Lemme just drop this screenshot here: 
![image](https://user-images.githubusercontent.com/13400422/54634583-858d9b80-4a50-11e9-9e96-859e34f8699c.png)

**Comment by Mewchild** [comment]
There have been situations in the past where some event has caused the entire server to slog to a standstill, the most prevalent one I can think of being the old Amber Slime outbreaks, in xenobiology. The server would process one tick every 30 seconds or so, and with no admins online, a vote was required to restart the round. This is what the vote is meant for, situations like that. Use common sense.

**Comment by TheDavestDave** [comment]
if you can make that screen shot appear everytime the vote button is press I will be happy, Sadly I am trying to figure out how to and failing badly

**Comment by Mewchild** [comment]
I'll reiterate my last line: Use Common Sense. I've only seen the vote restart used once, a single time, for a non-legit reason in my two years playing here. I don't think the issue is as big as it's being made out to be.

**Comment by SpadesNeil** [comment]
Everyone else beat me to it.

Here's a simple way to deal with bullshit votes. _Vote against it._ Don't just sit there and be like "lol this will never pass" because I'll just laugh when it passes.

If we see an organized effort to abuse the system intentionally, which has yet to happen, I will just issue bans based on the global rules.
